pkg/plugins/localfiles.go

```diff
@@ -33,7 +33,7 @@ func NewLocalFS(basePath string) LocalFS {
 // fileIsAllowed takes an absolute path to a file and an os.FileInfo for that file, and it checks if access to that
```

Comment:
```suggestion
// fileIsAllowed takes an absolute path to a file and an fs.FileInfo for that file, and it checks if access to that
```

Reply:
see https://github.com/grafana/grafana/pull/73587#discussion_r1321754124 - I'm going to back out the change so I won't change this either.